pandas/tests/io/json/test_pandas.py

```diff
@@ -1769,3 +1769,18 @@ def test_to_json_multiindex_escape(self):
             "\"(Timestamp('2017-01-23 00:00:00'), 'bar')\":true}"
         )
         assert result == expected
+
+    def to_json_series_of_objects(self):
```

Comment:
```suggestion
    def test_to_json_series_of_objects(self):
```
Having `test` in the name is necessary for the test to be discovered and run during CI